### PR TITLE
feat(relative-period): add relative-period to costQuerySet options

### PR DESCRIPTION
**Pull request by sulmoJ**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [ ] Not that difficult

### Type of Change
- [ ] New feature
- [x] Bug fixes
- [x] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [ ] mirinae
  - [ ] etc 

- [ ] Apps
  - [ ] storybook
  - [x] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description
I added the excludeToday option to distinguish between this month and last month.

### CODE REVIEW

File: apps/web/src/services/cost-explorer/cost-analysis/type.ts [hunk 1]
```diff
@@ -44,7 +45,11 @@ export interface Legend {
 }
 
 
-export type CostAnalysisPeriodType = typeof COST_ANALYSIS_PERIOD_TYPE[keyof typeof COST_ANALYSIS_PERIOD_TYPE];
+export type RelativePeriod = {
+    unit: OpUnitType;
+    value: number;
+    exclude_today: boolean;
```
Comment (sulmoJ) [inline apps/web/src/services/cost-explorer/cost-analysis/type.ts:51]: suggestion: I think includeToday is more understandable. Please fix it, sulmo

**Review by piggggggggy — APPROVED** [review]
Wow.. you did it!
I don't think it was easy.. LGTM!

File: apps/web/src/services/cost-explorer/cost-analysis/modules/CostAnalysisPeriodSelectDropdown.vue [hunk 2]
```diff
@@ -1,147 +1,128 @@
...
-    fixedPeriod: Period;
-}
-const props = withDefaults(defineProps<Props>(), {
-    fixedPeriod: () => getInitialDates(),
-});
-const emit = defineEmits<{(e: 'update', period: Period): void;
-}>();
 
 const today = dayjs.utc();
 interface PeriodItem {
```
Comment (yuda110) [inline apps/web/src/services/cost-explorer/cost-analysis/modules/CostAnalysisPeriodSelectDropdown.vue:30]: suggestion: How about this

```typescript
interface PeriodItem extends SelectDropdownMenu {
    period: ...
    ...
}
```

File: apps/web/src/services/cost-explorer/store/cost-analysis-page-store.ts [hunk 3]
```diff
@@ -30,23 +31,38 @@ export const useCostAnalysisPageStore = defineStore('cost-analysis-page', {
...
         groupBy: [],
         chartGroupBy: undefined,
-        period: getInitialDates(),
-        periodType: undefined,
+        period: convertRelativePeriodToPeriod({ unit: 'month', value: 5, exclude_today: true }),
+        relativePeriod: {
+            unit: 'month',
+            value: 5,
+            exclude_today: true,
+        },
```
Comment (yuda110) [inline apps/web/src/services/cost-explorer/store/cost-analysis-page-store.ts:39]: suggestion: 

```typescript
period: undefined,
relativePeriod: undefined
```

File: apps/web/src/services/cost-explorer/store/cost-analysis-page-store.ts [hunk 4]
```diff
@@ -30,23 +31,38 @@ export const useCostAnalysisPageStore = defineStore('cost-analysis-page', {
...
             this.groupBy = [];
             this.chartGroupBy = undefined;
-            this.period = getInitialDates();
-            this.periodType = undefined;
+            this.period = convertRelativePeriodToPeriod({ unit: 'month', value: 5, exclude_today: true });
+            this.relativePeriod = {
+                unit: 'month',
+                value: 5,
+                exclude_today: true,
+            };
```
Comment (yuda110) [inline apps/web/src/services/cost-explorer/store/cost-analysis-page-store.ts:65]: suggestion: here, too

```typescript
this.period = undefined;
this.relativePeriod = undefined;
```

File: apps/web/src/services/cost-explorer/cost-analysis/modules/CostAnalysisPeriodSelectDropdown.vue [hunk 5]
```diff
@@ -1,147 +1,128 @@
...
-    label: TranslateResult;
-    start: Dayjs;
-    end: Dayjs;
+    label: TranslateResult|Dayjs;
+    period: {
+        start: string
+        end: string;
+    };
+    relativePeriod?: RelativePeriod;
     enabled: Granularity[];
```
Comment (yuda110) [inline apps/web/src/services/cost-explorer/cost-analysis/modules/CostAnalysisPeriodSelectDropdown.vue:38]: suggestion: It seems like we can remove `enabled`.

**Review by yuda110 — COMMENTED** [review]
I've left quite a few comments. 😅
Thank you for your effort

**Review by yuda110 — APPROVED** [review]
LGTM!!!!!!

**Pull request merged**